### ISSUE TITLE
Set Active Tab of Workbook

### ISSUE THREAD

**Issue by kappaOne** [dhatim/fastexcel#130]
It would be helpful if you could add a feature to set the active tab of a workbook.

I would implement this by making the following changes to Workbook.java

`private int activeTab = 0; // global variable`

```
public void setActiveTab(int tab) {
    this.activeTab = tab;
}
```

And make the following change to line 187 in the private writeWorkbookFile method by changing
`"<workbookView activeTab=\"0\"/>" +`
to
`"<workbookView activeTab=\"" + activeTab + "\"/>" +`

**Comment by dorssar** [comment]
Maybe this should be called on the Sheet object:

sheet.setActive() or sheet.keepInActiveTab() could set the activeTab in workbook which can then write it to xml

**Comment by dorssar** [comment]
implemented in #124
